insonmnia/cgroups/cgroup_linux.go

```diff
@@ -83,19 +81,23 @@ func initializeControlGroup(name string, resources *specs.LinuxResources) (CGrou
 		// Create an empty cgroup, we update it later.
 		control, err = cgroups.New(cgroups.V1, cgroupPath, &specs.LinuxResources{})
 		if err != nil {
-			return nil, fmt.Errorf("failed to create parent cgroup %s: %v", name, err)
+			return nil, fmt.Errorf("failed to create cgroup %s: %v", name, err)
 		}
 	default:
-		return nil, fmt.Errorf("failed to load parent cgroup %s: %v", name, err)
+		return nil, fmt.Errorf("failed to load cgroup %s: %v", name, err)
 	}
 
 	if resources == nil {
 		return &cgroup{control, name}, nil
 	}
 
 	if err = control.Update(resources); err != nil {
-		return nil, fmt.Errorf("failed to set resource limit on parent cgroup %s: %v", name, err)
+		return nil, fmt.Errorf("failed to set resource limit on cgroup %s: %v", name, err)
 	}
 
 	return &cgroup{control, name}, nil
 }
+
+func initializeControlGroup(name string, resources *specs.LinuxResources) (CGroup, error) {
+	return newCGroup("/"+name, resources)
```

Comment:
Is there any reason not to do this into the `newCGroup`  function? Why we need an extra function just for appending a slash?